src/components/clusterDetail/ClusterCredentials.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Cluster } from '../../api/types';
+import { GridItem, TextContent, ButtonVariant, Button } from '@patternfly/react-core';
+
+type ClusterCredentialsProps = {
+  cluster: Cluster;
+};
+
+const ClusterCredentials: React.FC<ClusterCredentialsProps> = () => {
+  // TODO(jtomasek): Fetch cluster credentials data
+  // TODO(jtomasek):
+  return (
+    <GridItem>
+      <TextContent>
+        <dl className="cluster-detail__details-list">
+          <dt>Web Console URL</dt>
+          <dd>https://console-openshift-console.apps.clusterName.dev.domain.com</dd>
+          <dt>Username</dt>
+          <dd>kubeadmin</dd>
```

Comment:
Leftover?

Reply:
It is rather a placeholder, should I remove it? I don't think this gets rendered as we're not able to get to 'installed' state yet.